mypy/checkexpr.py

```diff
@@ -1485,7 +1485,15 @@ def check_call_expr_with_callee_type(
         )
         proper_callee = get_proper_type(callee_type)
         if isinstance(e.callee, (NameExpr, MemberExpr)):
-            self.chk.warn_deprecated_overload_item(e.callee.node, e, target=callee_type)
+            node = e.callee.node
+            if (node is None) and (member is not None) and isinstance(object_type, Instance):
```

Comment:
It is really strange that `ruff` / `black` do not catch this extra `()`.

I see that that's your prefered style, but we don't use them in mypy's codebase :)

Reply:
Okay, that's fine. I will get used to it.